.eslintrc.js

```diff
@@ -148,6 +148,15 @@ module.exports = {
       'no-prototype-builtins': 'off',
       'no-multi-spaces': 'off',
     },
+  }, {
+    files: ['test/**/*.ts'],
+    rules: {
+      // ignore the [_unusedVar, variable]
+      '@typescript-eslint/no-unused-vars': ['error', {
+        destructuredArrayIgnorePattern: '^_',
+        varsIgnorePattern: '[iI]gnore',
+      }],
+    },
```

Comment:
I saw one recommendation for that. Don't you think comments in parameter lists clutter the code? It also goes against typical JS conventions, that use `_` for de-structured assignment and nebulous functional args. It's such a common pattern that `eslint` uses `^_` throughout its examples: https://eslint.org/docs/rules/no-unused-vars#destructuredarrayignorepattern

Reply:
I'll turn off the rule and go back through and undo the code, but submit this should be re-considered.

Reply:
We can change the lint rules but that should happen in its own discussion thread / PR instead of here
Personally I would prefer to avoid declaring dummy variables as much as possible.
`/* ignore */` also has precedent as we have to put that in empty `catch` blocks by convention

Reply:
Earlier there was the suggestion to create a new Slack channel for this kind of discussions since #sequelize-help might not be the easiest. Maybe we should follow up on that

Reply:
`#sequelize-contributions`? (I was going to say `#sequelize-development` but could be confusing?)

Reply:
I asked Sacha if it would be worth creating #contribute: 

- Name: contribute
- Description: Contributing to Sequelize and Sequelize-CLI source code and documentation.

Padded with maintainers/contributors.  Just need a space to openly discuss ideas of the project with a lot of back-and-forth, and reduce the noise from those asking for help in #squelize-help.